src/main/kotlin/org/malachite/estella/services/RecruitmentProcessService.kt

```diff
@@ -53,4 +58,35 @@ class RecruitmentProcessService(@Autowired private val recruitmentProcessReposit
         return recruitmentProcessRepository.findAll().find { it.stages.contains(recruitmentProcessStage) }!!
     }
 
+    fun updateStagesList(jwt: String?, processId: Int, stagesList: List<String>) {
+        val userFromJWT = securityService.getHrPartnerFromJWT(jwt)
+        val process = getProcess(processId)
+        if (process.offer.creator.id != userFromJWT?.id) throw UnauthenticatedException()
+        val stages = updateStagesList(process.stages, stagesList.toListOfRecruitmentStage())
+        recruitmentProcessRepository.save(process.copy(stages = stages))
+    }
+
+    private fun updateStagesList(oldStages: List<RecruitmentStage>, newStages: List<RecruitmentStage>): List<RecruitmentStage> {
```

Comment:
Nie wiem czy bym nie zmienił nazwy jednej z tych dwóch funkcji. Bo dwie funkcje o takiej samej nazwie, ale wykonujące co innego wydaje mi się trochę mylące, ale to może tylko mi.

Reply:
Druga funkcja jest prywatna, więc nie będziemy jej używali poza tą klasą - ale fakt, ta nazwa została jakoś tak przed tym całym pierdzieleniem się z tym, więc pasuje ją zmienić